app/templates/email_templates/email_edit_file.html

```diff
@@ -0,0 +1,79 @@
+{% extends 'email_templates/email_response.html' %}
+{% block response_content %}
+    {% if not agency %}
+        {% if response_data.data_old['privacy'] != response_privacy.PRIVATE %}
+            <p>
+                A FILE in request <a href="{{ page }}">{{ request_id }}</a> has been edited.
+            </p>
+        {% else %}
+            <p>
+                A FILE in request <a href="{{ page }}">{{ request_id }}</a> is available for you to view
+            </p>
+            {% if response_data.response.request.requester.is_anonymous_requester %}
+                {% for key in response_data.get_file_link %}
```

Comment:
Using a for loop here is confusing, it suggests multiple files. There should be better way to do this, one that doesn't involve `key` (which really is the filename but someone looking at this in the future would only truly know that by checking where get_file_link is defined).